examples/Example/src/showcase/betterSwipeable/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Alert, StyleSheet, Text, View } from 'react-native';
+import { FlatList, RectButton } from 'react-native-gesture-handler';
+
+import { GmailStyleSwipeableRow } from './GmailStyleSwipeableRow';
+import { AppleStyleSwipeableRow } from './AppleStyleSwipeableRow';
+
+type DataRow = {
+  from: string;
+  when: string;
+  message: string;
+};
+
+const Row = ({ item }: { item: DataRow }) => (
+  <RectButton style={styles.rectButton} onPress={() => Alert.alert(item.from)}>
+    <Text style={styles.fromText}>{item.from}</Text>
+    <Text numberOfLines={2} style={styles.messageText}>
+      {item.message}
+    </Text>
+    <Text style={styles.dateText}>{item.when} ❭</Text>
+  </RectButton>
+);
+
+const SwipeableRow = ({ item, index }: { item: DataRow; index: number }) => {
+  if (index % 2 === 0) {
+    return (
+      <GmailStyleSwipeableRow>
+        <Row item={item} />
+      </GmailStyleSwipeableRow>
+    );
+  } else {
+    return (
+      <AppleStyleSwipeableRow>
+        <Row item={item} />
+      </AppleStyleSwipeableRow>
+    );
+  }
```

Comment:
```suggestion
  const SwipeableRow = index % 2 === 0
    ? GmailStyleSwipeableRow 
  	: AppleStyleSwipeableRow;
  return (
    <SwipeableRow>
      <Row item={item} />
    </SwipeableRow>
  );
```

Reply:
Done in f8b264c00123c2fc560986c6a80bf5c4d961de12